src/modules/control.cpp

```diff
@@ -2,60 +2,117 @@
 // Control Module
 //
 
+#include <stdint.h>
+
 #include "../pt/pt.h"
 
 #include "../hal/motor.h"
 #include "../hal/timer.h"
 
 #include "../modules/module.h"
 #include "../modules/control.h"
+#include "../modules/sensors.h"
 #include "../modules/parameters.h"
 
+#define INHALATION_OVERTIME 4 / 3
+#define HOLD_TIME           (150 MSEC)
```

Comment:
The guidance from Saurabh is to employ an inspiratory pause of 200 ms. Are you open to changing the name of this definition to something like INSPIRATORY_PAUSE_TIME?

Reply:
Yes, we can certainly make this 200 instead